slides/src/content/component.md

```diff
@@ -226,13 +226,13 @@ export class Child {
 
 ## Quiz (continued)
 
-1. Parent:  
```

Comment:
1. Brought in by accident (?)
2. The end-of-line double space is to force a line break - please use `<br>` instead.